Update XFF header documentation (#533)
kyma-project/istio: docs/user/00-10-overview-istio-controller.md

## X-Forwarded-For HTTP Header

The **X-Forwarded-For** (XFF) header is only supported on AWS clusters.

> The X-Forwarded-For header conveys the client IP address and the chain of intermediary proxies traversed by the request to reach the Istio Service Mesh.

Reviewer: ```suggestion
The XFF header conveys the client IP address and the chain of intermediary proxies that the request traversed to reach the Istio service mesh.
```
the rule about abbreviations is that once we explain what it stands for ([like here](https://github.com/kyma-project/istio/pull/533/files#diff-e1e09ad28d658e440e67eeaab48533f8ec8b17e7e34315d3642e91d111aa268fR34)), we can use it freely

The header might not include all IP addresses if an intermediary proxy does not support modifying the header.

> Due to [technical limitations of AWS Classic ELBs](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-proxy-protocol.html#proxy-protocol) when using an IPv4 connection the header does not include the public of the load balancer in front of the Istio Ingress Gateway.

Reviewer: ```suggestion
Due to [technical limitations of AWS Classic ELBs](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-proxy-protocol.html#proxy-protocol), when using an IPv4 connection, the header does not include the public IP of the load balancer in front of Istio Ingress Gateway.
```
should it be "public IP" or just "public"?

> Moreover, the Istio Ingress Gateway Envoy will not append the private IP address of the load balancer to the X-Forwarded-For header, effectively removing this information from the request.

Reviewer: ```suggestion
Moreover, Istio Ingress Gateway Envoy does not append the private IP address of the load balancer to the XFF header, effectively removing this information from the request.
```

## TLS termination

> The `istio-ingressgateway` Service creates a Layer 4 load balancer, that does not terminate TLS connections. Within the Istio Service Mesh,

Reviewer: ```suggestion
The `istio-ingressgateway` Service creates a Layer 4 load balancer, that does not terminate TLS connections. Within the Istio service mesh,
```
